OrbitCore/Capture.cpp

```diff
@@ -615,7 +608,9 @@ std::shared_ptr<CallStack> Capture::GetCallstack(CallstackID a_ID) {
 //-----------------------------------------------------------------------------
 void Capture::CheckForUnrealSupport() {
   GUnrealSupported =
-      GCoreApp->GetUnrealSupportEnabled() && GOrbitUnreal.HasFnameInfo();
+      GCoreApp != nullptr
```

Comment:
Nit: Maybe that's just me, but
`GCoreApp != nullptr && GCoreApp->GetUnrealSupportEnabled() && GOrbitUnreal.HasFnameInfo()`
feels more natural.